codecs/wp2/enc/wp2_enc.cpp

```diff
@@ -18,6 +19,7 @@ val encode(std::string image_in, int image_width, int image_height, WP2::Encoder
   }
 
   WP2::MemoryWriter memory_writer;
+  config.thread_level = emscripten_num_logical_cores() - 1;
```

Comment:
Why -1?

Reply:
Because in WebP2 thread_level is number of additional threads (where `0` is threads being disabled). Yes, it's unusual and I had to check :)

Reply:
Add comment pls :)

Reply:
Done.